driver/config.py

```diff
@@ -56,6 +57,7 @@ def load():
 		Config.MANAGEMENT_USERNAME = ConfigLoader._get_env_var_or_default('MANAGEMENT_USERNAME', default='admin@excelero.com')
 		Config.MANAGEMENT_PASSWORD = ConfigLoader._get_env_var_or_default('MANAGEMENT_PASSWORD', default='admin')
 		Config.SOCKET_PATH = ConfigLoader._get_env_var_or_default('SOCKET_PATH', default=Consts.DEFAULT_UDS_PATH)
+		Config.DRIVER_NAME = ConfigLoader._get_env_var_or_default('DRIVER_NAME', default=Consts.DRIVER_NAME)
```

Comment:
if `Consts.DRIVER_NAME` is now only the default value, please rename it to `Consts.DEFAULT_DRIVER_NAME` (also in `Consts.py`)

Reply:
Sorry I missed that one, I think this is fixed now and the sanity test has been updated accordingly